CMakeLists.txt

```diff
@@ -941,7 +941,7 @@ if (LLVM_ENABLE_DOXYGEN)
   message(STATUS "Doxygen: enabled")
 endif()
 
-if(SWIFT_BUILD_SYNTAXPARSERLIB OR SWIFT_BUILD_SOURCEKIT)
+if(SWIFT_BUILD_SYNTAXPARSERLIB OR SWIFT_BUILD_SOURCEKIT OR SWIFT_ENABLE_EXPERIMENTAL_CONCURRENCY)
```

Comment:
I'm guessing you'll need to teach `_Concurrency` to link against the (C) lib dispatch on Linux and Windows.